consensus/core/src/commit.rs

```diff
@@ -546,30 +546,29 @@ impl CommitInfo {
 }
 
 /// CommitRange stores a range of CommitIndex. The range contains the start (inclusive)
-/// and end (exclusive) commit indices and can be ordered for use as the key of a table.
-#[derive(Clone, Default, Debug, PartialEq, Eq, Serialize, Deserialize)]
-pub(crate) struct CommitRange(Range<CommitIndex>);
+/// and end (inclusive) commit indices and can be ordered for use as the key of a table.
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
```

Comment:
`Copy` can be derived as well, so we don't need to explicitly `clone()` CommitRange.

Reply:
Unfortunately `RangeInclusive` does not impl `Copy`